tdrs-frontend/src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import axios from 'axios'
 
-import { ConnectedRouter as Router } from 'connected-react-router'
+import { ReduxRouter as Router } from '@lagunovsky/redux-react-router'
```

Comment:
The replacement for `connected-react-router` has a very similar api and only requires a few additions to our root Router and Reducer below